src/argilla/client/api.py

```diff
@@ -479,4 +480,31 @@ def active_client() -> Argilla:
     return ArgillaSingleton.get()
 
 
+def whoami() -> None:
+    # Check the current user on the Argilla Server
+
+    from rich import print as rprint
+    from rich.markdown import Markdown
+
+    user = active_client()._user
```

Comment:
I think we can re-use the `__repr__` from the `active_client` directly.  Also you should use `logging` instead of `rprint`